Backoffice/Validator/Constraints/BooleanConditionValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use OpenOrchestra\ModelInterface\Form\DataTransformer\ConditionFromBooleanToBddTransformerInterface;
+
+/**
+ * Class BooleanConditionValidator
+ */
+class BooleanConditionValidator extends ConstraintValidator
+{
+    /**
+     * @param string $value
+     * @param Constraint $constraint
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (false === $this->validateCondition($value)) {
+            $this->context->buildViolation($constraint->message)
+                 ->addViolation();
+        }
+    }
+
+    /**
+     * @param string $columnWidth
+     *
+     * @return bool
+     */
+    protected function validateCondition($condition)
+    {
+        $is_boolean = true;
+        $encapsuledElements = array();
+        preg_match_all(
+            ConditionFromBooleanToBddTransformerInterface::GET_BALANCED_BRACKETS,
+            $condition,
+            $encapsuledElements
+        );
+        if(count($encapsuledElements[0]) > 0) {
```

Comment:
could not happen : preg_match_all will return at last an empty array for index 0 if pattern is correctly coded. Add it in test.